modules/allocating-additional-resources-to-data-science-users.adoc

```diff
@@ -0,0 +1,33 @@
+:_module-type: PROCEDURE
+
+[id='allocating-additional-resources-to-data-science-users_{context}']
+= Allocating additional resources to {productname-short} users
+
+[role='_abstract']
+As a cluster administrator, you can allocate additional resources to a cluster to support compute-intensive data science work. This support includes increasing the number of nodes in the cluster and changing the cluster's allocated machine pool.
+
+ifdef::self-managed[]
+For more information about allocating additional resources to an {openshift-platform} cluster, see link:https://docs.openshift.com/container-platform/4.13/machine_management/manually-scaling-machineset.html[Manually scaling a compute machine set].
+endif::[]
+
+ifndef::self-managed[]
```

Comment:
Does the appropriate version of these instructions appear upstream? I'm asking because there is no explicit upstream versus downstream conditionalization here, and I can't remember how (or if) it evaluates the 'self-managed' conditional statements.